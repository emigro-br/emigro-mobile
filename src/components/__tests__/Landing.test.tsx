import React from 'react';
import renderer from 'react-test-renderer';

import { Landing } from '@components/landing';

describe('<Landing />', () => {
  it('Renders "Hello World" message', () => {
    const tree = renderer.create(<Landing />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
