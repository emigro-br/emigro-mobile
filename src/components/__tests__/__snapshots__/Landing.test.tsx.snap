// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Landing /> Renders "Hello World" message 1`] = `
<Text>
  Hello World!
</Text>
`;
